src/plugins/dashboard/public/dashboard_actions/filters_notification_popover.tsx

```diff
@@ -59,8 +61,9 @@ export function FiltersNotificationPopover({ api }: { api: FiltersNotificationAc
     }
   }, [api, setDisableEditButton]);
 
-  const dataViews = useStateFromPublishingSubject(
-    api.parentApi?.dataViews ? api.parentApi.dataViews : new BehaviorSubject(undefined)
+  const [hasLockedHoverActions, dataViews] = useBatchedOptionalPublishingSubjects(
+    api.hasLockedHoverActions$,
+    api.parentApi?.dataViews
   );
 
   return (
```

Comment:
Is it possible to disable the Edit filters button when the dashboard is in view mode? 
<img width="713" alt="Screenshot 2024-10-22 at 9 08 06 AM" src="https://github.com/user-attachments/assets/4a8e5cb7-22cf-4717-b284-279240cb6438">

This can also lead to confusing UX for read-only users.


https://github.com/user-attachments/assets/f99ee143-a510-4844-b4d0-408ea6667e9c

Reply:
Addressed in [d429c69](https://github.com/elastic/kibana/pull/182535/commits/d429c6933a07207c1b7baf3b00748520b334d597)
<img width="1325" alt="Screenshot 2024-10-22 at 8 19 03 AM" src="https://github.com/user-attachments/assets/91693436-5db8-45ad-8100-d4c13c9d7c09">

Reply:
Can we wrap the entire footer with the `parentViewMode === 'edit'` so that we don't have the weird breaking line? Or just disable the button in view mode 🤔